uaa_bot/notifier.py

```diff
@@ -0,0 +1,112 @@
+from email.mime.text import MIMEText
+import os
+import smtplib
+import ssl
+
+from email_validator import validate_email, EmailNotValidError
+from jinja2 import Environment, PackageLoader, select_autoescape
+
+from uaa_bot import config, templates
+
+TEMPLATES = os.listdir(
+    os.path.join(os.path.dirname(os.path.realpath(__file__)), "./templates")
+)
+
+TEMPLATE_ENV = Environment(
+    loader=PackageLoader("uaa_bot", "templates"), autoescape=select_autoescape(["html"])
+)
+
+
+class Notifier:
```

Comment:
Possible nit:  shouldn't this inherit `object`?

```python
class Notifier(object):
```

Reply:
Not in python3

Reply:
D'oh!  Pandemic 🧠 strikes again...  I noticed it above elsewhere which triggered the thought.

Reply:
I have no historical context on what the proper format for simple class declaration.  Is `class MyClass:` the same inheritance as `class MyClass(object)` in python 3?

Reply:
yeah, in python 3, they're identical.

In python2, there were "new-style" and "old-style" classes, and "new-style" required the explicit inheritance to work. 
(new-style were introduced in python 2.2, so _a while_ ago)